tests/test_app/library/settings.py

```diff
@@ -18,6 +18,8 @@
 INSTALLED_APPS = [
     # Keep this above 'django.contrib.admin'
     "jazzmin",
+    "admin_numeric_filter",
```

Comment:
Best we add a comment here, something like

`# 3rd/4th party app support demonstrated by jazzmin, NOT needed for use of jazzmin`